packer/template.json

```diff
@@ -150,7 +150,7 @@
     {
         "type": "atlas",
         "only": ["vmware-iso"],
-        "artifact": "rackhd/rackhd",
+        "artifact": "heckj/rackhd",
```

Comment:
rackhd/rackhd is the office - this is work in progress, so was shimming in my personal repo so my experiments didn't break anything